apps/routes-api/src/app/services/routes.service.ts

```diff
@@ -0,0 +1,21 @@
+import {Route, RouteModel} from "../models/route.model"
+
+export class RoutesService{
+
+    async getRouteById(id: string): Promise<Route> {
```

Comment:
You should just add the update method, as the other ones are going to be added for other functionalities 🙇🏼